x-pack/plugins/fleet/server/saved_objects/migrations/to_v7_13_0.ts

```diff
@@ -23,3 +27,16 @@ export const migrateSettingsToV7130: SavedObjectMigrationFn<
 
   return settingsDoc;
 };
+
+export const migratePackagePolicyToV7130: SavedObjectMigrationFn<PackagePolicy, PackagePolicy> = (
+  packagePolicyDoc,
+  migrationContext
+) => {
+  // Endpoint specific migrations
+  // FIXME:PT remove `-OFF` from below once ready to be released
```

Comment:
@jfsiii , @nchaulet for now, I just turned this off below. This needs to run only when fleet-server is in the picture. Should I bring back the `xpack.fleet.agents.fleetServerEnabled` flag?

Reply:
Is the only place in which needs to fork based on that? Are there other options for detecting/inferring Fleet Server?

Reply:
I am okay to bring back the flag dependings on the timing of Fleet server for 7.13 we may have to hide some UI too, so as a temporary workaround it's probably our best solution

Reply:
@jfsiii I don't know the answers there.
For endpoint, I think this is the only branching we need in fleet - we already have security_solution specific flag in our code to handle one other branch.

@nchaulet thanks. I will add back in with another PR. I want this one to merge since it corrects the index property definition.